kubernetes-examples/snirouting_tls/postinstall.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/../../scripts/common.sh"
+
+KAFKA_TOOL_SUFFIX=".sh"
+if [ "$OS" = 'Darwin'  ]; then
+  if brew --prefix kafka 1>/dev/null 2>/dev/null; then
+     KAFKA_TOOL_SUFFIX=""
+  fi
+fi
+
+DNS_NAMES=$(${KUBECTL} get certificate -n kafka server-certificate -o json | jq -r '.spec.dnsNames | join(" ")')
+BOOTSTRAP=$(${KUBECTL} get certificate -n kafka server-certificate -o json | jq -r '.spec.dnsNames[0]')
+
+echo 'Please start "minikube tunnel" in another terminal.'
+
+while true
+do
+   EXTERNAL_IP=$(${KUBECTL} get service/kroxylicious-service -n kafka -o json | jq -r '.status.loadBalancer.ingress[0].ip // empty')
+   if [[ "${EXTERNAL_IP}" ]]; then
+      echo
+      echo "Found external IP ${EXTERNAL_IP} of load balancer service."
+      echo "Please add following link to your '/etc/hosts'."
+      echo "${EXTERNAL_IP} ${DNS_NAMES}"
```

Comment:
Having completely missed this, I believe @robobario did the same, can we make this stand out a bit more?
```suggestion
      echo -e  "\e[1;33mFound external IP ${EXTERNAL_IP} of load balancer service.\e[0m"
      echo -e  "\e[1;33mPlease add following link to your '/etc/hosts'.\e[0m"
      echo -e  "\e[1;33m${EXTERNAL_IP} ${DNS_NAMES}\e[0m"
```

Where 33 is yellow (see [Linux Shell Scripting Cookbook](https://www.oreilly.com/library/view/linux-shell-scripting/9781785881985/b0ddd805-aa79-441d-b5a7-380c66c7712d.xhtml)) but red or green could work too.